docs/content/config.md

```diff
@@ -17,37 +17,53 @@ schema:
 
 # Where should the generated server code go?
 exec:
-  layout: follow-schema
-  dir: graph/generated
-  package: generated
-
-# Enable Apollo federation support
-federation:
-  filename: graph/federation.go
   package: graph
-  version: 2
-  options:
-    computed_requires: true
+  layout: single-file # Only other option is "follow-schema," ie multi-file.
+
+  # Only for single-file layout:
+  filename: graph/generated.go
+
+  # Only for follow-schema layout:
+  # dir: graph
+  # filename_template: "{name}.generated.go"
+
+  # Optional: Maximum number of goroutines in concurrency to use per child resolvers(default: unlimited)
+  # worker_limit: 1000
+
+# Uncomment to enable federation
+# federation:
+#   filename: graph/federation.go
+#   package: graph
+#   version: 2
+#   options:
+#     computed_requires: true
```

Comment:
Not sure about commenting this out. FE People use Apollo on the frontend and are not always aware that that they are using Apollo _**federation**_ directives when they are using a single backend, but want to dip their does into gqlgen and have it "just work"